src/datasources/alerts-api/tenderly-api.service.ts

```diff
@@ -49,7 +49,7 @@ export class TenderlyApi implements IAlertsApi {
         },
       });
     } catch (error) {
-      this.httpErrorFactory.from(error);
+      throw this.httpErrorFactory.from(error);
```

Comment:
Are the error messages documented on Tenderly? We need to be sure that we are not leaking any sensitive data while mapping the error from the HttpErrorFactory (as it can forward the error message to the other layers).

Reply:
As discussed, this will be changed before released in a follow-up PR to ensure no "leaks".